test/lib/ufe/testUINodeGraphNode.py

```diff
@@ -53,14 +53,18 @@ def setUp(self):
         # Open ballset.ma scene in testSamples
         mayaUtils.openGroupBallsScene()
 
+        self.messageHandler = mayaUtils.TestProxyShapeUpdateHandler('|transform1|proxyShape1')
+
         # Clear selection to start off
         cmds.select(clear=True)
 
+    def tearDown(self):
+        self.messageHandler.terminate()
+
     # Helper to avoid copy-pasting the entire test
     def doPosAndSizeTests(self, hasFunc, setFunc, getFunc, cmdFunc):
-        initialUpdateCount = cmds.getAttr('|transform1|proxyShape1.updateId')
```

Comment:
Note; a fix was already merged in dev. You might want to merge dev in your branch.

Reply:
Yes, I have merged with latest dev as I need the recent leak fix.